lightify/__init__.py

```diff
@@ -962,6 +966,21 @@ def _next_seq(self):
             self.__seq = (self.__seq + 1) % 256
             return self.__seq
 
+    def set_loglevel(self, debugLevel):
```

Comment:
Please consider changing `debugLevel` to `log_level` or just `level`, because `debugLevel` suggests it's actually `logging.DEBUG`.  Also please add a docstring.

```suggestion
    def set_loglevel(self, level):
    """ set the log level
    
    :param level: log level (from logging module)
    """
```